test/integration/production-swcminify/test/index.test.js

```diff
@@ -103,9 +103,6 @@ describe.skip('Production Usage with swcMinify', () => {
       expect(
         serverTrace.files.some((file) => file.includes('node_modules/sharp'))
       ).toBe(false)
-      expect(
-        serverTrace.files.some((file) => file.includes('react.development.js'))
-      ).toBe(false)
```

Comment:
Are we expecting react.development.js in production builds now?

Reply:
Yeah it's required to be included for ESM support as it seems only the last `module.exports` is used for detecting named exports so if we don't include it, it fails 

x-ref: https://vercel.slack.com/archives/CGU8HUTUH/p1635892319339400?thread_ts=1635842152.280700&cid=CGU8HUTUH
x-ref: https://github.com/nodejs/cjs-module-lexer#moduleexports-reexport-assignment